### ISSUE TITLE
Logo lighting on Rudder is not consistent

### ISSUE THREAD

**Issue by TTpilot45** [headwindsim/A339X#54]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2023-01-28T19:21:29+00:00",
    "ref": "refs/heads/main",
    "sha": "305cc44dcbbdc74fd45fbc876e1c3a7ee5b05c53",
    "actor": "Revyn112",
    "event_name": "manual",
    "pretty_release_name": "main:305cc44d",
    "version": "v0.4.0-dev.305cc44"
}
```

### Describe the bug

logo lighting on the rudder is not consistent on both sides

### Expected behavior

Same splash of light on either side

### Steps to reproduce

Trun on nav lights at night and observe logo on the rudder.

**Comment by Normolo** [comment]
I've attached an example of this
![image](https://user-images.githubusercontent.com/53821637/224095896-da133edb-8e77-4d6b-927a-187b82bed3d2.png)
![image](https://user-images.githubusercontent.com/53821637/224096030-7b214602-d75f-41c0-883f-3e2afbe19968.png)

**Comment by Revyn112** [comment]
fixed #42a8a76243d71b6b3d05db9f0f68aaa1133acf5b will be updated in next stable version
